ArtOfIllusion/src/artofillusion/SplineMeshEditorWindow.java

```diff
@@ -758,8 +758,7 @@ void subdivideCommand()
     param = new double [usize+usplitcount][vsize][numParam];
     for (i = 0; i < param.length; i++)
       for (j = 0; j < param[i].length; j++)
-        for (int k = 0; k < param[i][j].length; k++)
```

Comment:
To be clear, for the small mesh, we are talking differences of a couple tenths of a millisecond for the entire method to return. When we move up to a 32X64 mesh being subdivided, it's more like a couple hundredths of a second, but at that scale other things are causing bigger issues. The System.arrayCopy() seems to create more junk, as well. Other factors, possibly GC, inflated the Standard Deviation on that run up to 95% of the mean. At that scale of subdivision, my UI becomes unresponsive.